CONTRIBUTING.md

```diff
@@ -135,3 +137,42 @@ This can easily be done with the `-s` command line option to append this automat
 ```sh
 git commit -s -m 'This is my commit message'
 ```
+#### Code Review of the Pull Request
+
+In Nephio Project uses [Prow](https://docs.prow.k8s.io/) to among many other tasks test and manage the Pull Requests. Prow also allows to interact with PR trough ['slash' commands](https://prow.nephio.io/command-help)
+
+##### The Code Review Process
+The **author** submits a PR
+- Step 0: [Prow bot](https://github.com/apps/nephio-prow) assigns reviewers and approvers for the PR based on [OWNERS files](https://www.kubernetes.dev/docs/guide/owners/) please note that single repository can have multiple OWNERS files, bot will choose reviewers and approvers from file nearest to changed code 
+- Step 1: Prow tests the PR
+If the author is not (yet) a member of Nephio GitHub organization Prow will add label 'needs-ok-to-test' which will prevent Prow from running tests on it. This is a security measure as many tests allow to execute arbitrary code, create objects in infrastructure and so on. To allow the tests to run any member of organization can use command
+/ok-to-test in comment, then tests will run. Status of their execution will be visible on PR page itself or if one wants to see logs, previous runs etc those are available on [Prow's Dashboard](https://prow.nephio.io/)
+Definions of those tests are either in local [inrepoconfig](https://docs.prow.k8s.io/docs/inrepoconfig/) file .prow.yaml or in [central Prow configuration](https://github.com/nephio-project/test-infra/tree/main/prow/config)
```

Comment:
These three lines from markdown appear on same line after rendering and thus reduces readability. I believe the intention was to have sub-bulleted points?

Reply:
Same for below points. The markdown line format is not in sync with the final rendered text where all the text appears on the same line.

Reply:
Thanks for all suggestions! In this case those were just line breaks to fit better in GitHub's code viewer. Do you think it's unreadable? It's one paragraph, not bullet points.

Reply:
If that was the intention then it is fine. Just thought of apprising.